packages/create-cloudflare/src/helpers/files.ts

```diff
@@ -35,19 +35,15 @@ export const writeJSON = (path: string, object: object, stringifySpace = 2) => {
 };
 
 // Probes a list of paths and returns the first one that exists
-// If one isn't found, throws an error with the given message
-export const probePaths = (
-	paths: string[],
-	errorMsg = "Failed to find required file."
-) => {
+// If one isn't found, throws an error
+export const probePaths = (paths: string[]) => {
 	for (const path of paths) {
 		if (existsSync(path)) {
 			return path;
 		}
 	}
 
-	crash(errorMsg);
-	process.exit(1); // hack to make typescript happy
```

Comment:
instead of crashing/existing I changed this so that it simply throws and the caller can decide if/how to catch and deal with the exception

without this change I couldn't have used the function to check the existence of paths in `getAppDirPath`

I hope this is fine 🙂

Reply:
Definitely better to throw.
But I think we should go further and simply return `null` if none is found.
Then the caller can decide whether to throw or recover and we don't have to pass in an error message then either.

Reply:
ah yeah, good point! 🙂👍

Reply:
done 🙂